src/vm/tests/simple_apply_eval.rs

```diff
@@ -929,6 +929,36 @@ fn test_lets() {
         .for_each(|(program, expectation)| assert_eq!(expectation.clone(), execute(program)));
 }
 
+#[test]
+fn test_2239_merge() {
```

Comment:
Can we have a more descriptive test name?  Naming code after GH issues is a pet peeve of mine, because unless you're willing to trawl through the GH issue list, you won't know what the context for the test is.

Reply:
Sure, added more description. The advantage of including the issue number in the test name is that it makes running the single test much easier. `cargo test <test-name>` doesn't "regex match", it just does substring matching, so if test names are substrings of one another, it's impossible to just run one test.